pkg/codegen/template_helpers.go

```diff
@@ -241,6 +241,10 @@ func toStringArray(sarr []string) string {
 	return `[]string{"` + strings.Join(sarr, `","`) + `"}`
 }
 
+func toSingleLine(s string) string {
+	return strings.ReplaceAll(s, "\n", "")
```

Comment:
The build is failing because go is not able to find the `strings` package

Reply:
Probably because travis is using [go 1.11 ](https://travis-ci.com/deepmap/oapi-codegen/builds/130332379#L195) but replaceAll was introduced in [go 1.12](https://github.com/Codewars/codewars.com/issues/1861#issuecomment-507417558)

Reply:
Yeah good spot. I've switched over to using the go 1.11 compatible `strings.Replacer`.